src/shared.js

```diff
@@ -4,9 +4,9 @@
  */
 
 /**
- * A value for strings that signals a Part to clear its content
+ * A value for parts that signals a Part to clear its content
  */
-export const nothingString = '__nothing-lit-html-server-string__';
+export const nothing = {};
```

Comment:
I think I’ve changed my mind about this. Can we revert the value back to a string but keep the name? Referential equality can break across realms, but not strings